decompiler/util/data_decompile.cpp

```diff
@@ -604,6 +604,17 @@ goos::Object nav_mesh_nav_control_arr_decompile(
                                                file, TypeSpec("nav-control"), 288);
 }
 
+goos::Object xz_height_map_data_arr_decompile(const std::vector<LinkedWord>& words,
+                                              const std::vector<DecompilerLabel>& labels,
+                                              int my_seg,
+                                              int field_location,
+                                              const TypeSystem& ts,
+                                              const std::vector<std::vector<LinkedWord>>& all_words,
+                                              const LinkedObjectFile* file) {
+  return decomp_ref_to_inline_array_guess_size(words, labels, my_seg, field_location, ts, all_words,
+                                               file, TypeSpec("vector4b"), 4);
```

Comment:
discussed before, the type is indeed likely a `(pointer int8)` but I'm not sure if that is something we can easily support in the decompiler right now or if that should be a new issue.